doc/changelog.md

```diff
@@ -298,7 +450,7 @@
 
 * Binary distributions for Linux are now based on `manylinux_2_28` instead of `manylinux_2014`.
   As a result, Catalyst will only be compatible on systems with `glibc` versions `2.28` and above
-  (e.g. Ubuntu 20.04 and above).
+  (e.g., Ubuntu 20.04 and above).
   [(#663)](https://github.com/PennyLaneAI/catalyst/pull/663)
 
 <h3>Bug fixes</h3>
```

Comment:
801 needs information that this is about the runtime:
```
* An issue in the Lightning backend for the Catalyst runtime has been fixed that would only compute
  approximate probabilities when implementing mid-circuit measurements. As a result, low shot numbers 
  would lead to unexpected behaviours or projections on zero probability states.
  Probabilities for mid-circuit measurements are now always computed analytically.
```

Reply:
thanks, this is much more informative!